python/src/aac/plugins/gen_design_doc/gen_design_doc_impl.py

```diff
@@ -93,8 +95,17 @@ def _get_output_file_extension(template_filespec: str) -> str:
 
 
 def _get_and_prepare_definitions_by_type(parsed_definitions: list[Definition], aac_type: str) -> list[dict]:
+    def get_definition_structure_with_required_fields(interface_definition: Definition):
```

Comment:
It's cleaner to keep a space around inner functions.

```suggestion

    def get_definition_structure_with_required_fields(interface_definition: Definition):
```